alibi_detect/saving/_sklearn/loading.py

```diff
@@ -0,0 +1,29 @@
+import os
+from pathlib import Path
+from typing import Union
+
+import joblib
+from sklearn.base import BaseEstimator
+
+
+def load_model(filepath: Union[str, os.PathLike],
+               load_dir: str = 'model',
+               ) -> BaseEstimator:
+    """
+    Load scikit-learn (or xgboost) model. Models are assumed to be a subclass of :class:`~sklearn.base.BaseEstimator`.
+    This includes xgboost models following the scikit-learn API
+    (see https://xgboost.readthedocs.io/en/latest/python/python_api.html#module-xgboost.sklearn).
+
+    Parameters
+    ----------
+    filepath
+        Saved model directory.
+    load_dir
+        Name of saved model folder within the filepath directory.
```

Comment:
Not quite sure of the difference between the args? Is `filepath` the saved detector directory and the `load_dir` the location within the detector directory? Why not just have `filepath`?

Reply:
Fair point. This is really only for consistency with the `_tensorflow.loading.load_model` function. It has these two separate kwarg's for use in the legacy `load_detector` here: https://github.com/SeldonIO/alibi-detect/blob/6fa17081735c11275dc7e551ddc672674e679b93/alibi_detect/saving/_tensorflow/loading.py#L275-L276.

Happy to remove now for sklearn if you think best?

Reply:
Removed in [c3fd225](https://github.com/SeldonIO/alibi-detect/pull/642/commits/c3fd2256c92bc2ff96b8d28dde946d90cde8cefa) :)